paimon-web-ui/src/views/cdc/components/list/index.tsx

```diff
@@ -20,10 +20,11 @@ import { useTable } from './use-table'
 
 export default defineComponent({
   name: 'ListPage',
-  setup() {
+  emits:['cdcJobSubmit'],
+  setup(props,ctx) {
```

Comment:
If you do not use this props, please use an '_' as the parameter name. 
```suggestion
  setup(_, ctx) {
```